frontend/src/scenes/billing/BillingLimitInput.tsx

```diff
@@ -13,87 +12,106 @@ import { billingProductLogic } from './billingProductLogic'
 export const BillingLimitInput = ({ product }: { product: BillingProductV2Type }): JSX.Element | null => {
```

Comment:
Can we change the name of this component to BillingLimitSection or something like that? It's no longer just the input.

Reply:
Yep, good suggestion.